dist/api_data.py

```diff
@@ -557,6 +557,12 @@ def __ge__(self, other):
     Config('debug_mode', '', r'''
         control the settings of various extended debugging features''',
         type='category', subconfig=[
+        Config('background_compact_expire_time', '86400', r'''
+               how long the compact stats for a file should be tracked for before expiring.''',
```

Comment:
I think it would be better to frame this more like the dhandle scan option `close_idle_time`, the current description exposes more of the internal implementation than I think is ideal.